ocs/rook.py

```diff
@@ -7,10 +7,12 @@
 """
```

Comment:
Please change the file name.
This has nothing to do with rook

Reply:
This one is up for change in next PR